### PR TITLE
Update cats to 1.4.0

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

**Comment by kubukoz** [comment]
I just realised cats doesn't support Scala 2.10 since 1.3.0, so we'll need to remove 2.10 from the versions this module is published to...

File: modules/cats/build.sbt
Comment (ruippeixotog) [inline modules/cats/build.sbt]: Nitpick: can you leave the versions defined in-place as before? Since it is only used twice near one another, I think it becomes more readable if we just write the same thing twice.

**Review by ruippeixotog — COMMENTED** [review]
Hi @kubukoz, thanks for the PR! In fact, #412 will bring the end of PureConfig for Scala 2.10, so this module won't need to be defined differently. This looks good to me, but let's wait until #412 is merged so that we can just update the version.

**Comment by kubukoz** [comment]
Good to know! I'll get back to this one after #412 is closed then.

File: modules/cats/build.sbt
Comment (kubukoz) [reply modules/cats/build.sbt]: Sure, if that works better for you I'll inline :)

**Comment by kubukoz** [comment]
I see #412 was merged so I'll get back to this one :)

**Comment by kubukoz** [comment]
Should be good to go.

File: build.sbt
Comment (leifwickland) [inline build.sbt]: Nitpick: why this change? It appears to be unnecessary and inconsistent with the surrounding code.

**Review by leifwickland — APPROVED** [review]
Thanks for the contribution, @kubukoz!

File: build.sbt
Comment (kubukoz) [reply build.sbt]: Yeah, it's a leftover from when I was removing a 2.10 build. I'm rolling it back

**Review by jcazevedo — APPROVED** [review]
Thanks for this, @kubukoz!

**Pull request merged**
